app/src/main/res/values/strings.xml

```diff
@@ -102,6 +102,7 @@
   <string name="ratio_error_invalid_chars">Please write a ratio that consists of digits separated by colons (e.g. 1:2 or 1:2:3).</string>
   <string name="ratio_error_invalid_format">Please enter a valid ratio (e.g. 1:2 or 1:2:3).</string>
   <string name="ratio_error_invalid_colons">Your answer has two colons (:) next to each other.</string>
+  <string name="ratio_error_invalid_size">Number of terms is less than required terms.</string>
```

Comment:
Please add a description for this and other new strings.

Reply:
done